### PR TITLE
Handle name field being required for Location resources

### PR DESCRIPTION
**Pull request by pkmitre**
This pull request

1. Supports creation of VRDR records where there is no data for Location resource name fields where a non-blank name string is required using "BLANK" as a placeholder
2. Supports consumption of similar records
3. Incidentally cleans up tests a bit bit not reloading test files when already loaded

### CODE REVIEW

File: VRDR.Tests/DeathRecord_Should.cs [hunk 1]
```diff
@@ -2135,6 +2132,10 @@ public void Set_DispositionLocationName()
         {
             SetterDeathRecord.DispositionLocationName = "Bedford Cemetery";
             Assert.Equal("Bedford Cemetery", SetterDeathRecord.DispositionLocationName);
+            SetterDeathRecord.DispositionLocationName = "";
```
Comment (saulakravitz) [inline VRDR.Tests/DeathRecord_Should.cs:2135]: should be "BLANK" (or similar)...."" triggers a warning about empty strings.

File: VRDR.Tests/DeathRecord_Should.cs [hunk 2]
```diff
@@ -2135,6 +2132,10 @@ public void Set_DispositionLocationName()
...
+            Assert.Null(SetterDeathRecord.DispositionLocationName);
+            SetterDeathRecord.DispositionLocationName = null;
```
Comment (saulakravitz) [inline VRDR.Tests/DeathRecord_Should.cs:2137]: should be "BLANK" (or similar)...."" triggers a warning about empty strings.

File: VRDR.Tests/DeathRecord_Should.cs [hunk 3]
```diff
@@ -2135,6 +2132,10 @@ public void Set_DispositionLocationName()
...
+            Assert.Null(SetterDeathRecord.DispositionLocationName);
```
Comment (saulakravitz) [inline VRDR.Tests/DeathRecord_Should.cs:2138]: Not sure this is the right test.

File: VRDR.Tests/DeathRecord_Should.cs (hunk 1 above)
Comment (pkmitre) [reply VRDR.Tests/DeathRecord_Should.cs:2135]: Look at the code for the property, it handles this correctly putting BLANK in the FHIR representation.

File: VRDR.Tests/DeathRecord_Should.cs (hunk 2 above)
Comment (pkmitre) [reply VRDR.Tests/DeathRecord_Should.cs:2137]: Look at the code for the property, it handles this correctly putting BLANK in the FHIR representation.

**Review by saulakravitz — APPROVED** [review]

**Pull request merged**